ambari-server/src/main/java/org/apache/ambari/server/api/services/ServiceConfigVersionService.java

```diff
@@ -22,16 +22,32 @@
 import java.util.Map;
 
 import javax.ws.rs.GET;
+import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.Context;
 import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.UriInfo;
 
-import org.apache.ambari.annotations.ApiIgnore;
+import org.apache.ambari.annotations.SwaggerOverwriteNestedAPI;
 import org.apache.ambari.server.api.resources.ResourceInstance;
+import org.apache.ambari.server.controller.ServiceConfigVersionResponse;
 import org.apache.ambari.server.controller.spi.Resource;
 
+import org.apache.http.HttpStatus;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiImplicitParam;
+import io.swagger.annotations.ApiImplicitParams;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+@Api(value = "Configurations", description = "Endpoint for configuration-specific operations")
```

Comment:
Is this intentionally the same as the one in the parent?

Reply:
yep, `ServiceConfigVersion` is available through `Configurations` without additional params and there is only one function within the service so I think it make sense to show them in one group. 

service endpoint: `/api/v1/clusters/local/configurations/service_config_versions`